library/core/src/num/f128.rs

```diff
@@ -1292,7 +1287,6 @@ impl f128 {
     /// ```
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
```

Comment:
```suggestion
    #[unstable(feature = "f128", issue = "116909")]
    #[rustc_const_unstable(feature = "f128", issue = "116909")]
```
Seems better just for consistency, and in case the intrinsic gets added.

`is_finite` will then also need the attribute.